### ISSUE TITLE
The response is the same as the request if the request is null

### ISSUE THREAD

**Issue by mcorbelli** [jamauro/method#13]
How come, if a method doesn't have the response (because in some cases it doesn't have to have it), on the client I get the request that is sent to the method as the response?

**Comment by jamauro** [comment]
Hmm, I have a guess but if you could provide steps to reproduce that would help diagnose. Thanks!

**Comment by mcorbelli** [comment]
I solved by modifying this line of code. instead of input as the last condition, i return null. Can this be a solution? At the moment i have the package locally

`input = (run ? (beforePipeline.includes(func) ? input : afterPipeline.includes(func) ? runResult : result) : result) || null;`

**Comment by jamauro** [comment]
I think it's on the right track but will break other scenarios. If you can provide more detail and show me how you're defining the method, then we can come up with a robust solution.

**Comment by mcorbelli** [comment]
Oh very simple, this is the server method i call inside run (createMethod). As you see i don't need a return value. With the latest version of the library the value of the response the client receives is the request i send to the method. 

<img width="597" alt="SCR-20240918-ruva" src="https://github.com/user-attachments/assets/109dd622-4d18-4fd4-9cc6-dddb91bbe66a">

**Comment by jamauro** [comment]
I see. Well I imagine if you put a `return` after `await category.delete()` then no other changes would be required.

**Comment by mcorbelli** [comment]
No, because the return would always be null or undefined. In my opinion anyway it would be an interesting modification to improve the package. What happened to me is that all the before I declared are used to add information to the request, each with a specific purpose. And I found myself this information on the client without me doing anything (some information was sensitive)

**Comment by mcorbelli** [comment]
I meant if I hooked a return without a value, of course. but, as said before, I don't think all methods deserve to have a return

**Comment by jamauro** [comment]
Ok, thanks for the feedback. I plan to address this soon.

**Comment by jamauro** [comment]
Thanks for reporting. This has been fixed in `v.1.7.3`. I'm going to close this issue but if you experience any problems, feel free to reopen.
